Makefile

```diff
@@ -747,6 +750,11 @@ npm-cache:
 	touch -c node_modules
 
 node_modules:
+ifneq ($(NODEJS),)
+ifneq ($(NODE_VERSION),$(shell $(NODEJS) --version | sed 's/\(v[0-9]*\.[0-9]*\)\.[0-9]*/\1/'))
```

Comment:
ah we need to duplicate "$" in makefile, so the full line is:
```
ifneq ($(NODE_VERSION),$(shell $(NODEJS) --version | awk -F. '{ print $$1 FS $$2 }'))
```
Also, are we supposed to work with node 0.12 ?